.gitignore

```diff
@@ -8,7 +8,6 @@ bin
 bin
 .classpath
 .project
-.settings
```

Comment:
Why did you remove that folder from the gitignore?

Reply:
Because I've set tons of prettier rules inside so everyone uses the same.